### PR TITLE
fix(cli): increase testRunner.jest.setupTimeout from 2 min to 5 min

### PR DESCRIPTION
**Pull request by remarkablemark**
## Description

- This pull request addresses the issue described here: 

In this pull request, I have updated the `.detoxrc.js` config generated by the CLI to match the value in the [documentation](https://wix.github.io/Detox/docs/config/testRunner/#testrunnerjestsetuptimeout-number):

```
testRunner.jest.setupTimeout: 300000 (5 minutes)
```

This fixes timeout issues on CI:

```
FAIL e2e/starter.test.ts
  ● Test suite failed to run

    Exceeded timeout of 120000ms while setting up Detox environment
```

See https://github.com/remarkablemark/react-native-cli-quickstart

---

> _For features/enhancements:_
 - [ ] I have added/updated the relevant references in the [documentation](https://github.com/wix/Detox/tree/master/docs) files.

> _For API changes:_
 - [ ] I have made the necessary changes in the [types index](https://github.com/wix/Detox/blob/master/detox/index.d.ts) file.

### CODE REVIEW

File: detox/local-cli/init.js [hunk 1]
```diff
@@ -76,7 +76,7 @@ function createDefaultConfigurations() {
         config: 'e2e/jest.config.js',
       },
       jest: {
-        setupTimeout: 120000,
+        setupTimeout: 300000,
```
Comment (asafkorem) [inline detox/local-cli/init.js:79]: @remarkablemark thanks for the contribution! 
@noomorph we need to decide whether we should replace the default value in the docs or here.

@remarkablemark - in general, you should be able to configure the test-runner and set a jest runner with five minutes setup-timeout or whatever is needed for your project.

Comment (remarkablemark) [reply detox/local-cli/init.js:79]: Thanks for the response. Happy to close the PR if the docs get updated.

Comment (noomorph) [reply detox/local-cli/init.js:79]: @remarkablemark thanks for your keen eye! 👍 Yes, we updated the docs.

If you have other suggestions, feel welcome to create an issue or a pull request.

**Comment by noomorph** [comment]
Fixed by https://github.com/wix/Detox/commit/85047ee1165e6dda6a0439551e68ead654bfcf04

**Pull request closed without merging**